.circleci/config.yml

```
@@ -19,7 +21,34 @@ jobs:
+    steps:
+      - checkout
+      - run:
+          name: Install YAMLlint
+          command: |
+            sudo apt-get update
+            sudo apt-get install -y npm
+            sudo npm install -g yaml-lint
+      - run:
+          name: Lint YAML files
+          command: |
+            yaml-lint **/*.yaml || true  
```

yamllint

[error] trailing spaces

(trailing-spaces)